content/docs/user-guide/components/reference/gradients/image-gradient.md

```diff
@@ -26,38 +26,61 @@ configuration options, such as tiling, scaling, and custom sampling types.
+|  Bicubic | The `Bicubic` filter smooths out the image by requesting sixteen points in a grid around a requested pixel and then performing Catmull-Rom interpolation between the points.<br><br>Because this uses non-linear interpolation, there are no noticeable plus-shaped artifacts in the smoothed data. `Bicubic` sampling is the best choice when quality is needed over performance. | 16x | Great | ![Image Gradient using bicubic interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bicubic.png) |
 
-![Image Gradient using point sampling](/images/user-guide/components/reference/gradients/image-gradient-component-point.png)
+## Editing an image
 
-#### Bilinear
+To edit an existing image, press the **Edit** button on the Image Gradient component or selecting the Image Gradient component's icon in the **Component Switcher** in the viewport. This will enter Paint Brush mode. See the [Paint Brush](/docs/user-guide/components/reference/paintbrush/paintbrush) documentation for more details on how to use the Paint Brush.
 
-The `Bilinear` filter can be used to smooth out the image by requesting four points in a grid around a requested
-pixel and then performing interpolation between the points.
+After editing is complete, end the Paint Brush mode by pressing **Esc**, pressing the **Done** button on the Image Gradient component, or by selecting a different component's icon in the **Component Switcher** in the viewport. At this point, the image changes will be saved, re-processed by the Asset Processor, and reloaded when processing is complete.
 
-![Image Gradient using bilinear interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bilinear.png)
+The **Save Mode** determined where the image will be saved.
+| Save Mode | Description |
+| - | - |
+| `Save As...` | Prompts for a save location every time the image is saved. |
+| `Auto Save` | Prompts for a save location the first time the image is saved after loading the level, but on every subsequent save automatically overwrites the image in that location. |
+| `Auto Save With Incrementing Names` | Automatically saves the image with an incrementing number at the end of the name, and only prompts for a save location if there is already an existing image with that name.<br><br>For example, if the initially-selected image is `image_gsi.tif`, this will save it as `image_gsi.0000.tif`, then as `image_gsi.0001.tif`, then as `image_gsi.0002.tif`, etc. |
```

Comment:
```suggestion
| `Auto Save With Incrementing Names` | Automatically saves the image with an incrementing number at the end of the name and only prompts for a save location if there is already an existing image with that name.<br><br>For example, if the initially-selected image is `image_gsi.tif`, this will save it as `image_gsi.0000.tif`, then as `image_gsi.0001.tif`, then as `image_gsi.0002.tif`, etc. |
```